src/context/OrderContext.tsx

```diff
@@ -0,0 +1,29 @@
+import { OrderCreateDto } from 'dtos/OrderCreateDto';
+import { createContext, useReducer } from 'react';
+import { orderReducer } from './orderReducer';
+
+const initialState = { lineItems: [] };
+
+const OrderContext = createContext<{
+  state: OrderCreateDto;
+  dispatch: React.Dispatch<any>;
+}>({
```

Comment:
put this type def into an interface so that prettier could format it properly 😄

Reply:
done